src/templates/witnesspubkeyhash/input.js

```diff
@@ -1,9 +1,44 @@
 // {signature} {pubKey}
 
-var pkh = require('../pubkeyhash/input')
+var bscript = require('../../script')
+var typeforce = require('typeforce')
+
+function isCompressedCanonicalPubKey (pubKey) {
+  return bscript.isCanonicalPubKey(pubKey) && pubKey.length === 33
+}
+
+function check (script, strict) {
```

Comment:
Issue here,  `check` is usually `check(script, allowIncomplete)`,  not `check(script, strict)`... I don't think this is OK?